content/webapp/pages/pages/[pageId].tsx

```diff
@@ -120,25 +120,15 @@ export const getServerSideProps: GetServerSideProps<
     ? context.resolvedUrl
     : undefined;
 
-  const pageDocument = await fetchPage(client, pageId);
+  const pageDocument = await fetchPage(
+    client,
+    pageId,
+    isSiteSection(siteSection) || siteSection === 'orphan'
```

Comment:
would it simplify things to add `'orphan'` to the `siteSectionList`?

Reply:
As discussed I've left comments to explain the logic https://github.com/wellcomecollection/wellcomecollection.org/pull/11283/commits/eb6dec4d332732c0e5743320dd99b4032436d07e